data/router_test.go

```diff
@@ -3,102 +3,88 @@ package data
 import (
 	"testing"
 
-	"github.com/magiconair/properties/assert"
-	db2 "github.com/tendermint/tm-db"
-
-	"github.com/Oneledger/protocol/data/balance"
-	"github.com/Oneledger/protocol/data/chain"
-	"github.com/Oneledger/protocol/data/fees"
-	"github.com/Oneledger/protocol/data/governance"
-	"github.com/Oneledger/protocol/data/keys"
 	"github.com/Oneledger/protocol/storage"
+	"github.com/magiconair/properties/assert"
+	"github.com/tendermint/tm-db"
 )
 
 const (
-	balanceType = "b"
-	governType  = "g"
-	feeType     = "f"
+	testType = "t"
 )
 
 var (
 	chainstate      *storage.ChainState
-	balanceStore    *balance.Store
```

Comment:
The reason why you removed these stores was because they weren't the correct type I'm assuming.

Reply:
yeah, I found this unit test was outdated since we added ExtStore interface to this part, and I just made a simpler unit test to fix this.